src/types.d.ts

```diff
@@ -48,7 +48,52 @@ declare global {
         total?: number,
         annotationCollection?: string,
         modules?: Module[]
-    
+    }
+
+    interface Colors {
+        forceMode: string,
+        primary: string,
+        secondary: string,
+        accent: string
+    }
+
+    interface ConnectorViewPanel {
+        id: number,
+        options?: Option_1_ConnectorViewPanel | Option_2_ConnectorViewPanel | Option_3_ConnectorViewPanel
+    }
+
+    type Option_1_ConnectorViewPanel = { 
```

Comment:
I think we should refine this a bit more. The types for different connector options should be more specific. A connector is in fact a real existing component that sits inside of a view (tab). For the sake of modularity we should push towards well-defined APIs (props/events) for each component. This means that we should define specifc types and interfaces for those views. For example a `MetadataView` component should be configured by a `MetadataViewConfigOptions` interface.

Reply:
I have worked in my previous two commits to address the refinement of `Config` interface.

Reply:
As discussed, please remove all non TextAPI Types and Interfaces for now and focus just on refactoring to Pinia store. We will deal with the correct typing of configs and their dependencies later.

Reply:
@paulpestov yes, I just did it in my last commit:
Additionally in my last commit I did the following:
- removed the `Config Vuex Module` folder 
- used `Config Pinia Store` in some other components
- used only 'TextAPI' types in 'Config' Pinia store
- removed unused code